src/loaders/lottie/tvgLottieModel.cpp

```diff
@@ -207,7 +207,7 @@ float LottieLayer::remap(float frameNo)
 
 LottieComposition::~LottieComposition()
 {
-    if (!initiated) delete(root->scene);
+    if (!initiated && root) delete(root->scene);
```

Comment:
initialized shouldn't be true if the root is null. So it doesn't need to change this.

Reply:
The case where parse() returns false is when json is invalid or there is no root layer. When initialized is false, there may be cases where the root layer is null. (1313 line in tvgLottieParser.cpp)
Without this change, root->scene will be called with root being null.

Reply:
ok, got it. thanks